lib/Factory/UserFactory.php

```diff
@@ -35,10 +35,12 @@ public function from($params) {
 	}
 
 	protected function from_id(int $id) {
-		return $this->build(get_user_by('id', $id));
+		$wp_user = get_user_by('id', $id);
+
+		return $wp_user ? $this->build($wp_user) : null;
 	}
 
-	protected function from_post_obj(object $obj) : CoreInterface {
+	protected function from_user_object(object $obj) : CoreInterface {
```

Comment:
The `object` type hint is only supported in PHP 7.2+. So I guess we can’t use it yet.

Reply:
Removed.